packages/orchestration/src/utils/zoe-tools.js

```diff
@@ -167,12 +171,13 @@ export const makeZoeTools = (
         paymentKwr,
       );
       console.debug(depositResponse);
-    },
-  );
+    });
 
   return harden({
     localTransfer,
     withdrawToSeat,
   });
 };
-/** @typedef {ReturnType<typeof makeZoeTools>} ZoeTools */
+
+/** @typedef {ReturnType<typeof makeZoeTools>} ZoeToolsHost */
+/** @typedef {GuestInterface<ZoeToolsHost>} ZoeTools */
```

Comment:
are we always going to have the unqualified name be the guest? I don't recall if we've made that decision.

Consider not defining `ZoeTools` and letting the guest places use `GuestInterface<ZoeToolsHost>`